src/test/java/hw3/tests/BaseTest.java

```diff
@@ -0,0 +1,50 @@
+package hw3.tests;
+
+import hw3.po.Browser;
+import hw3.po.HomePage;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public class BaseTest {
+
+    protected final String URL = "https://jdi-testing.github.io/jdi-light/index.html";
+    protected final String login = "Roman";
+    protected final String password = "Jdi1234";
+    protected final String username = "ROMAN IOVLEV";
```

Comment:
Parameters should be in the properties file. Please read requirements for the HW